sys/include/net/ng_pktbuf.h

```diff
@@ -213,9 +201,12 @@ void ng_pktbuf_stats(void);
 bool ng_pktbuf_is_empty(void);
 
 /**
- * @brief   Resets the whole packet buffer
+ * @brief   Checks if the implenation's internal invariants still uphold
+ *
+ * @return  true, the packet buffer is sane.
+ * @return  false, the packet buffer is insane.
  */
-void ng_pktbuf_reset(void);
```

Comment:
Tell me what to write apart from "initializes the packet buffer" and I'll do it :>